test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -115,17 +115,25 @@ def setup
         sub_tax_id: '987-65-4321'
       }
     }
-    @apple_pay_network_token = network_tokenization_credit_card(
-      '4895370015293175',
+    @apple_pay_shared_network_token_options = {
       month: 10,
       year: Time.new.year + 2,
       first_name: 'John',
       last_name: 'Smith',
       verification_value: '737',
-      payment_cryptogram: 'abc1234567890',
```

Comment:
The apple pay payment method will always have a cryptogram but it's up to the gateway adapter to not send it. So the payment method when creating it should have a payment_cryptogram